test.js

```diff
@@ -114,3 +116,98 @@ test.cb('should not skip empty directories', t => {
 	stream.pipe(vinylAssign({extract: true})).pipe(unzipper);
 	stream.end();
 });
+
+test.cb(`when 'options.mtimes' is specified, should override files' actual mtimes`, t => {
```

Comment:
I prefer only using template string when I actually want to interpolate something. Here you can just escape the `'` instead.

Reply:
That would require three escaped single quotes. Which I'll be happy to change. Or, just so that all options have been considered, how do you feel about surrounding the string with double quotes instead? No escaping needed. Will the linter complain? Let me know what you want.